packages/iframe-coordinator/src/elements/frame-router.ts

```diff
@@ -272,7 +272,9 @@ export default class FrameRouterElement extends HTMLElement {
     if (!hostUrlObject.hash) {
       hostUrlObject.pathname = stripTrailingSlash(hostUrlObject.pathname);
       if (window.location.hash) {
-        hostUrlObject.pathname += "/";
+        if (hostUrlObject.pathname !== "/") {
+          hostUrlObject.pathname += "/";
+        }
```

Comment:
The previous code assumed we needed to add a `/` because we would always strip a `/` on line 273, however if the original `pathname` is `/`, the URL object will not change it to an empty string on line 273. Instead, it is preserved as `/`, in which case we should not re-add the `/`. 

We have future roadmap work planned to remove this sort of processing entirely, but it would be too disruptive for consuming apps to do right now.